aiven/client/connection_info/kafka.py

```diff
@@ -52,7 +68,15 @@ def from_service(
             raise ConnectionInfoError(
                 "Cannot format kafka connection info for service type {service_type}".format_map(service)
             )
-
+        try:
+            find_component(
+                service["components"],
+                kafka_authentication_method="certificate",
+            )
```

Comment:
You have two `find_component` calls with different arguments with the same `kafka_authentication_method="certificate"`. Why not keep just one?

Reply:
I believe a standalone `find_component` checking only the `kafka_authentication_method` checks if the authentication method is enabled, while the `find_component` checking both `kafka_authentication_method` and `route` checks that the authentication method is enabled in the route specified?

Reply:
Ah, yeah, you are right. Thanks.